packages/components/src/components/teaser/teaser.test.ts

```diff
@@ -0,0 +1,126 @@
+import { expect, fixture, html } from '@open-wc/testing';
+import type SdTeaser from './teaser';
+
+describe('<sd-teaser>', () => {
+  it('renders default values correctly', async () => {
+    const el = await fixture<SdTeaser>(html`<sd-teaser></sd-teaser>`);
+
+    expect(el.variant).to.equal('white');
+    expect(el.breakpoint).to.equal(448);
+    expect(el.inset).to.equal(false);
+  });
+
+  it('renders assigned values correctly', async () => {
```

Comment:
We need tests for the breakpoints.

Something like:

default breakpoint
1. set width of component < breakpoint
2. check if state is vertical
3. set width of component > breakpoint
4. check if state is horizontal

manual breakpoint
1. set width of component < breakpoint
2. check if state is vertical
3. set width of component > breakpoint
4. check if state is horizontal